docs/getting-started/apple/screenshots.md

```diff
@@ -1,8 +1,258 @@
+
+## Setting Up _snapshot_
+
+1. Create a new UI Test target in your Xcode project (See the top part of [this article](https://krausefx.com/blog/run-xcode-7-ui-tests-from-the-command-line))
+1. Run `snapshot init` in your project folder
+1. Add the ./SnapshotHelper.swift file to your UI Test target (You can move the file anywhere you want)
+1. (Objective C only) Add the bridging header to your test class.
+    - `#import "MYUITests-Swift.h"`
+    - The bridging header is named after your test target with -Swift.h appended.
+1. In your UI Test class, click the `Record` button on the bottom left and record your interaction
+1. To take a screenshot, call the following between interactions
+    - Swift: `snapshot("01LoginScreen")`
+    - Objective C: `[Snapshot snapshot:@"01LoginScreen" waitForLoadingIndicator:YES];`
+1. Add the following code to your `setUp()` method:
+
+**Swift**
+
+```swift
+let app = XCUIApplication()
+setupSnapshot(app)
+app.launch()
+```
+
+**Objective C**
+
+```objective-c
+XCUIApplication *app = [[XCUIApplication alloc] init];
+[Snapshot setupSnapshot:app];
+[app launch];
+```
+
+If you have _fastlane_ installed, it's easy to give _snapshot_ a try. First clone the _fastlane_ repo, head over to the _snapshot_ example project, and then run `fastlane snapshot`
+
+```
+$ git clone https://github.com/fastlane/fastlane
+
+$ cd fastlane/snapshot/example
+
+$ fastlane snapshot
+```
+
+![assets/snapshot.gif](assets/snapshot.gif)
+
+# Decorate Screenshots with Device Frames
+
+_frameit_ helps you beautify your screenshots with devices frames and text by running one simple command. It provides support for:
+
+- Multiple device types
+- Portrait and landscape orientations
+- Black and silver devices
+- Setting a background color and decorating the image with text
+
+![frameit results](assets/frameit-results.png)
+
+## Dependencies
+
+### Installing ImageMagick
+
+_frameit_ depends on a tool called `imagemagick` to do image manipulation. The easiest way to install it is through [homebrew](http://brew.sh/):
+
+```sh
+brew install libpng jpeg imagemagick
+```
+
+### Troubleshooting ImageMagick
+
+If you have installed _imagemagick_ but are seeing error messages like:
+
+```sh
+mogrify: no decode delegate for this image format `PNG'
+```
+
+You may need to reinstall and build from source. Run:
+
+```sh
+brew uninstall imagemagick; brew install libpng jpeg; brew install imagemagick --build-from-source
+```
+
+### Setting Up Device Frames
+
+_fastlane_ uses device frames provided by Apple which need to be downloaded and installed separately for legal reasons. Running `fastlane frameit setup` can walk you through the process.
+
+`**TODO:** Update this when we start distributing frames ourselves.`
+
+```
+$ fastlane frameit setup
```

Comment:
The output below is shown on the user's machine anyway, I don't think we have to have them in the docs.